pkg/tsdb/elasticsearch/client/client.go

```diff
@@ -188,15 +189,21 @@ func (c *baseClientImpl) ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearch
 	clientRes, err := c.executeBatchRequest("_msearch", queryParams, multiRequests)
 	if err != nil {
 		status := "error"
+		errorSource = backend.ErrorSourceDownstream
 		if errors.Is(err, context.Canceled) {
 			status = "cancelled"
 		}
 		lp := []any{"error", err, "status", status, "duration", time.Since(start), "stage", StageDatabaseRequest}
 		if clientRes != nil {
```

Comment:
I think something here is not correct. If we don't get a `clientRes` we would now report the error as downstream, but I think then it should be plugin, right? Because not getting `clientRes` is likely due to misconfigured plugins.